demos/python_demos/llm_python_backend/README.md

```diff
@@ -0,0 +1,57 @@
+# Llama demo with python node {#ovms_demo_python_llama}
+
+## Build image
+
+From the root of the repository run:
+
+```bash
+git clone https://github.com/openvinotoolkit/model_server.git
+cd model_server
+make python_image MEDIAPIPE_DISABLE=0 PYTHON_DISABLE=0
```

Comment:
Can we remove those parameters? It should be obvious. There's no way to build that image without MP and Python.